docs/core/project-sdk/msbuild-props.md

```diff
@@ -159,14 +276,194 @@ The `PackRelease` property is similar to the [PublishRelease](#publishrelease) p
+
+When set to `true`, the `ApiCompatEnableRuleAttributesMustMatch` property enables the validation rule that checks if attributes match. The default is `false`.
+
+```xml
+<PropertyGroup>
+  <ApiCompatEnableRuleAttributesMustMatch>true</ApiCompatEnableRuleAttributesMustMatch>
+</PropertyGroup>
+```
+
+### ApiCompatEnableRuleCannotChangeParameterName
+
+When set to `true`, the `ApiCompatEnableRuleCannotChangeParameterName` property enables the validation rule that checks whether parameter names have changed in public methods. The default is `false`.
+
+```xml
+<PropertyGroup>
+  <ApiCompatEnableRuleCannotChangeParameterName>true</ApiCompatEnableRuleCannotChangeParameterName>
+</PropertyGroup>
+```
+
+### ApiCompatExcludeAttributesFile
+
+The `ApiCompatExcludeAttributesFile` item specifies the path to a file that contains attributes to exclude in [DocId](../../csharp/language-reference/xmldoc/index.md#id-strings) format.
+
+```xml
+<ItemGroup>
+  <ApiCompatExcludeAttributesFile Include="ApiCompatExcludedAttributes.txt" />
+  <ApiCompatExcludeAttributesFile Include="ApiCompatBaselineExcludedAttributes.txt" />
+</ItemGroup>
+```
+
+### ApiCompatGenerateSuppressionFile
+
+The `ApiCompatGenerateSuppressionFile` property specifies whether to generate a compatibility suppression file.
+
+```xml
+<PropertyGroup>
+  <ApiCompatGenerateSuppressionFile>true</ApiCompatGenerateSuppressionFile>
+</PropertyGroup>
+```
+
+### ApiCompatPermitUnnecessarySuppressions
+
+The `ApiCompatPermitUnnecessarySuppressions` property specifies whether to permit unnecessary suppressions in the suppression file.
+
+The default is `false`.
+
+```xml
+<PropertyGroup>
+  <ApiCompatPermitUnnecessarySuppressions>true</ApiCompatPermitUnnecessarySuppressions>
+</PropertyGroup>
+```
+
+### ApiCompatPreserveUnnecessarySuppressions
+
+The `ApiCompatPreserveUnnecessarySuppressions` property specifies whether to preserve unnecessary suppressions when regenerating the suppression file. When an existing suppression file is regenerated, its content is read, deserialized into a set of suppressions, and then stored in a list. Some of the suppressions might no longer be necessary if the incompatibility has been fixed. When the suppressions are serialized back to disk, you can choose to keep *all* the existing (deserialized) expressions by setting this property to `true`.
+
+The default is `false`.
+
+```xml
+<PropertyGroup>
+  <ApiCompatPreserveUnnecessarySuppressions>true</ApiCompatPreserveUnnecessarySuppressions>
+</PropertyGroup>
+```
+
+### ApiCompatRespectInternals
+
+The `ApiCompatRespectInternals` property specifies whether `internal` APIs should be checked for compatibility in addition to `public` APIs.
+
+```xml
+<PropertyGroup>
+  <ApiCompatRespectInternals>true</ApiCompatRespectInternals>
+</PropertyGroup>
+```
+
+### ApiCompatSuppressionFile
+
+The `ApiCompatSuppressionFile` item specifies the path to one or more suppression files to read from.
```

Comment:
It would be good to mention that by default (when not setting this item) "project-directory/CompatibilitySuppressions.xml" is used and read from (if it exists).